ResetPasswordBundle/src/Persistence/Repository/PasswordResetRequestRepositoryTrait.php

```diff
@@ -1,58 +1,67 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace SymfonyCasts\Bundle\ResetPassword\Persistence\Repository;
 
-use SymfonyCasts\Bundle\ResetPassword\Model\PasswordResetRequest;
+use Symfony\Component\Security\Core\User\UserInterface;
+use SymfonyCasts\Bundle\ResetPassword\Model\PasswordResetRequestInterface;
 
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver <weaverryan@gmail.com>
+ */
 trait PasswordResetRequestRepositoryTrait
 {
-    public function createPasswordResetRequest(object $user, \DateTimeInterface $expiresAt, string $selector, string $hashedToken): PasswordResetRequest
-    {
-        return new PasswordResetRequest(
-            $user,
-            $expiresAt,
-            $selector,
-            $hashedToken
-        );
-    }
-
-    public function getUserIdentifier(object $user): string
+    //@todo implement method or remove it
```

Comment:
It IS implemented! I just forgot to put `return` at the front of all of it :D

Reply:
done